packages/calypso-products/src/plans-list.tsx

```diff
@@ -384,6 +385,7 @@ const getPlanPersonalDetails = (): IncompleteWPcomPlan => ( {
 			// pay attention to ordering, shared features should align on /plan page
 			FEATURE_CUSTOM_DOMAIN,
 			FEATURE_HOSTING,
+			WPCOM_FEATURES_GLOBAL_STYLES,
```

Comment:
Note that this could use a feature flag check if you want to avoid putting this PR on hold:

```suggestion
			isEnabled( 'limit-global-styles' ) ? WPCOM_FEATURES_GLOBAL_STYLES : null,
```

Reply:
I'll leave on hold, that way I don't need to revert anything after.

Reply:
I'd strongly recommend to add the feature flag and revert later, so we can ask folks to test everything in the CfT by simply enabling a feature flag.